modules/ossm-rn-fixed-issues.adoc

```diff
@@ -41,6 +45,16 @@ This is fixed by using the Kiali SA to fetch the cluster version. This also allo
 
 * https://issues.redhat.com/browse/OSSM-2335[OSSM-2335] Dragging the mouse pointer over the Traces scatterchart plot sometimes caused the Kiali console to stop responding due to concurrent backend requests.
 
+* https://issues.redhat.com/browse/OSSM-2221[OSSM-2221] Previously, gateway injection in the `ServiceMeshControlPlane` namespace was not possible because the `ignore-namespace` label was applied to the namespace by default. 
++
+When creating a v2.4 control plane, the namespace no longer has the `ignore-namespace` label applied, and gateway injection is possible.
++
+In the following example, the `oc label` command removes the `ignore-namespace` label from a namespace in an existing deployment:
++
+`oc label namespace <istio_system> maistra.io/ignore-namespace-`
```

Comment:
This command needs to be formatted as a standard, terminal code block: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output